orca-retrofit/src/main/groovy/com/netflix/spinnaker/orca/retrofit/RetrofitConfiguration.groovy

```diff
@@ -34,40 +36,41 @@ import org.springframework.context.annotation.Scope
 import org.springframework.core.Ordered
 import org.springframework.core.annotation.Order
 import retrofit.RestAdapter.LogLevel
-import retrofit.client.Client
-import retrofit.client.OkClient
+
+import java.util.concurrent.TimeUnit
 
 @Configuration
 @CompileStatic
-@Import(OkHttpClientConfiguration)
+@Import(OkHttp3ClientConfiguration)
 @EnableConfigurationProperties
 class RetrofitConfiguration {
 
-   @Value('${okHttpClient.connectionPool.maxIdleConnections:5}')
+   @Value('${okHttp3Client.connectionPool.maxIdleConnections:5}')
    int maxIdleConnections
 
-   @Value('${okHttpClient.connectionPool.keepAliveDurationMs:300000}')
+   @Value('${okHttp3Client.connectionPool.keepAliveDurationMs:300000}')
    int keepAliveDurationMs
 
-   @Value('${okHttpClient.retryOnConnectionFailure:true}')
+   @Value('${okHttp3Client.retryOnConnectionFailure:true}')
    boolean retryOnConnectionFailure
```

Comment:
I'm not so sure we want to change the namespacing of these configs, I feel like it'd be really hard for our OSS users to track down potential regressions in config.

Reply:
what happens if we have two `@ConfigurationProperties` classes that use the same `prefix=okHttpClient`?